Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/PhotoUriManager.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.compose.jetsurvey.survey
+
+import android.content.ContentValues
+import android.content.Context
+import android.net.Uri
+import android.provider.MediaStore
+
+/**
+ * Manages the creation of photo Uris. The Uri is used to store the photos taken with camera.
+ */
+class PhotoUriManager(private val appContext: Context) {
+
+    private val photoCollection by lazy {
+        MediaStore.Images.Media.getContentUri(MediaStore.VOLUME_EXTERNAL_PRIMARY)
+    }
+
+    private val resolver by lazy { appContext.contentResolver }
+
+    // Uri used to save photos taken with the camera
+    var uri: Uri? = null
+        get() = field
+        private set(value) {
+            field = value
+        }
```

Comment:
Is this needed? Looks unnecessary?

Reply:
I want to make sure that this isn't set from outside the class. That only PhotoUriManager controls the setting. 
Is there a better way to achieve this?